metadata_annotation.ipynb

```diff
@@ -0,0 +1,2307 @@
+{
```

Comment:
<blockquote>the <code>lookup</code> class,</blockquote><p>a <code>lookup</code> class,</p><p> </p><blockquote>To sum up, it has the following functions:</blockquote><p>I think we can remove this and just immediately use <code>:</code> after "annotating" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>